CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Braintree iOS SDK Release Notes
 
-## unreleased
+## 5.10.0 (2022-06-06)
```

Comment:
```suggestion
## unreleased
```
We should leave this as "unreleased." We have an automated release script that generates this timestamp.

Reply:
Actually this may be the result of an auto merge.